lidar_litev3hp_sensor2/lidar_litev3hp_sensor2.ino

```diff
@@ -0,0 +1,77 @@
+/*------------------------------------------------------------------------------
```

Comment:
This file is a nearly identical duplicate of the one above. The only functional difference is this line
```
ros::Publisher pub_distance( "sensor2_data", &distance_msg);
char frameid[] = "/laser_lite_v3hp_sensor1";		char frameid[] = "/laser_lite_v3hp_sensor2";
```
Is there a way for us to make the change from sensor1 vs sensor2 done with arguments instead of having a duplicate file?